src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
+
+- Added antiAffinity (configuration)[/docs/self-hosted/administration/configuration/#userpodaffinity] for PODs with label `dev.okteto.com/affinity`
+- Add external resource icons
+- Build credential helpers
+- Remove load time on GitHub redeploy dialog
+
+### Bug Fix
+
+- When injecting host alises to go through internal network to buildkit, registry and api, don't duplicate entries for same IP. Instead, pass the list of hostnames for the IP
+- Delete button is now disabled in the personal namespace
+- Restore k3s support
+- Fix faulty error case on deployment with external resource
+- Fix faulty progressing state on deployment with external resource
+- Disable CTA while destroying all
+- Catch all non `/error/{code}` requests
+- Make env var values editable on deploy/redeploy
+- Keep ingress class on wake if ingress class is different than "sleeping"
+- Fix service unavailable when destroy all
+- Fix divert snippet configuration
+- Filter repositories results by search term
+- Fix pointer cursor in staged log lines
+- Autopreviews are now being deleted when PR is merged or closed
+- Align endpoints and repos on `ResourceDetails`
+- Remove leftover configuration from the cert-manager days
+- Fix logged data on SSE connection failure
+- Fix typo in OPENID_RESOURCE envvar for ADFS
```

Comment:
"Fixed issue in the OpenID configuration that prevented using    ADFS as the identity provider"